### PR TITLE
Update message for UI in case failed payment publish

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

New test case got in repo in parallel with revised exception handling / logging and we need to respond nicely to caseworkers. This PR addresses this edge case and fixes currently broken pipeline

P.S. Exception message is provided by BA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java [hunk 1]
```diff
@@ -258,6 +259,15 @@ private ProcessResult createNewCase(
...
             }
+        } catch (PaymentsPublishingException exception) {
+            log.error(
+                "Failed to send update to payment processor for {} exception record {}",
+                configItem.getService(),
+                exceptionRecord.id,
+                exception
+            );
+
+            throw new CallbackException("Payment references cannot be processed", exception);
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java]:
```suggestion
            throw new CallbackException("Payment references cannot be processed. Please try again later.", exception);
```

File: src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java [hunk 2]
```diff
@@ -793,17 +793,19 @@ void should_return_error_if_publishing_payment_command_failed() throws Exception
...
                     USER_ID
-                );
+                ),
+            CallbackException.class
+        );
 
         // then
-        assertThat(result.getErrors()).containsExactly("Internal error. " + paymentException.getMessage());
-        assertThat(result.getWarnings()).isEmpty();
+        assertThat(exception.getMessage()).isEqualTo("Payment references cannot be processed");
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java]:
```suggestion
        assertThat(exception.getMessage()).isEqualTo("Payment references cannot be processed. Please try again later.");
```

**Comment by rkondratowicz** [comment]
build passed, github not showing it
